.craft.yml

```diff
@@ -0,0 +1,10 @@
+---
+github:
+  owner: getsentry
+  repo: sentry-webpack-plugin
+changelogPolicy: simple
```

Comment:
So JFYI, this line means that `craft` will fail on new releases that don't have the corresponding changelog entry. The changelog file should look like this: https://github.com/getsentry/sentry-cordova/blob/master/CHANGELOG.md

Reply:
That's fine, I'll update the changelog